core/kong-gateway/kong.production.yml

```diff
@@ -63,6 +63,48 @@ services:
         config:
           allowed_iss: [ "http://172.17.0.1:7000/auth/realms/diva-kc-realm" ]
 
+  - name: "Service-Management"
+    url: "http://service-management:3004/"
+    routes:
+      - name: services-route
+        paths:
+          - "/services/"
+        protocols:
+          - "http"
+          - "https"
+        strip_path: true
```

Comment:
Should be `false`

Reply:
agree, so is it in UM config and UM works